developerportal/apps/common/models.py

```diff
@@ -26,5 +28,14 @@ class BasePage(Page):
         related_name="+",
     )
 
+    #  A single place to store/define cache keys that need invalidation
+    # when the object is saved
```

Comment:
We could do this differently and use redis more directly as a cache, in order to get wildcard invalidation or tagged sets of keys to invalidate